src/Annotations/Annotation.cpp

```diff
@@ -441,16 +442,16 @@ bool rawStringIsCorrectlyFormatted(const std::string &rawAnnotationString) {
   return true;
 }
 
-Annotation *generateAnnotation(const std::string &rawAnnotationString) {
-  if (!rawStringIsCorrectlyFormatted(rawAnnotationString)) {
+Annotation *
+Annotation::generateAnnotation(const std::string &rawAnnotationString) {
```

Comment:
I assume that the auto-formatter is responsible for this, but it looks bad - this probably shouldn't need to be on two lines. I'd consider renames here to get this back on one line. For example, the parameter could be called "rawAnnoString", "rawAnnoStr", or even just "rawAnno" (since its type is string)